core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/AttributeComponent.scala

```diff
@@ -269,6 +269,14 @@ trait AttributeComponent {
       }
     }
 
+    def findUniqueStringsByNameQuery(attrName: AttributeName) = {
+      filter(rec =>
+        rec.namespace === attrName.namespace &&
+        rec.name === attrName.name &&
+        rec.valueString.isDefined
+      ).map(_.valueString).distinct
+    }
```

Comment:
after trying a few approaches, I really wasn't happy with the size of the SQL query, coupled with the amount of work we had to do in Scala to get the results we wanted. Here's a more optimized query.

This generates SQL of:
```
select distinct `value_string` from `WORKSPACE_ATTRIBUTE` where ((`namespace` = 'tag') and (`name` = 'tags')) and (`value_string` is not null)
```